client/signup/steps/mlb-theme-selection/style.scss

```diff
@@ -0,0 +1,7 @@
+.mlb-themes__terms h2 {
+	font-size: 120%;
```

Comment:
We use `px` for font sizes in Calypso; I know it's not the best solution, but we should aim for consistency first. Also we tend to target class names rather than elements.